recipes/vulkan-loader/all/conanfile.py

```diff
@@ -0,0 +1,94 @@
+import os
+import glob
+from conans import ConanFile, tools, CMake
+
+
+class VulkanLoaderConan(ConanFile):
+    name = "vulkan-loader"
+    description = "Khronos official Vulkan ICD desktop loader for Windows, Linux, and MacOS."
+    topics = ("conan", "vulkan", "loader", "desktop", "gpu")
+    homepage = "https://github.com/KhronosGroup/Vulkan-Loader"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "Apache-2.0"
+    exports_sources = ["CMakeLists.txt"]
+    settings = "os", "arch", "build_type", "compiler"
+    generators = "cmake", "cmake_find_package"
+    short_paths = True
+    options = {
+        "with_wsi_xcb": [True, False],
+        "with_wsi_xlib": [True, False],
+        "with_wsi_wayland": [True, False],
+        "with_wsi_directfb": [True, False],
+    }
+    default_options = {
+        "with_wsi_xcb": False,
+        "with_wsi_xlib": False,
+        "with_wsi_wayland": False,
+        "with_wsi_directfb": False,
```

Comment:
it doesn't honor default values of upstream CMakeLists: https://github.com/KhronosGroup/Vulkan-Loader/blob/a148cdd490417324bf7db925ac2ff0f4e4cafd61/CMakeLists.txt#L121

Reply:
I don't think enabling `with_wsi_wayland` is a good idea because there is no conan package for Wayland and CMake will use Wayland libraries from the system.